### ISSUE TITLE
Flickity Card Slider Animation 

### ISSUE THREAD

**Issue by bgabot89** [metafizzy/flickity#661]
Hi,

I'm currently working with the flickity API for one of my freelance projects and I had a question regarding the usage of a flickity card animation.

I made a codepen example of what i replicated onto my project: https://codepen.io/bgabot89/pen/awQRay

Basically, I created a stack of cards by iterating through each card and assigning them a different translateX css property. This allows me to format the slides into a "3D card stack". The cards are formatted correctly but the biggest problem i'm having is the animation when moving from slide-to- slide. In the codepen example, the entire card stack is moving from the farthest right or left-side part of the page.

Here is an animation gif of what I need to happen:

![flickity_slider_example_small](https://user-images.githubusercontent.com/14283168/33250055-1ff0fe1a-d2e3-11e7-978f-950e47461b43.gif)

Basically, i need the slides to move to their correct positions without there being so much movement. I can't have the entire card stack move from the farthest right/left part of the page, like in the codepen example.  I need the slides to have a smooth transition from slide-to-slide like in the mockup gif.

I think the ".flickity-slider" element is preventing me from achieving this effect because it's making a full translateX for the entire flickity slide container.

Is there anyway to replicate the effect that I want through flickity? 

Thanks

**Comment by desandro** [comment]
> Is there anyway to replicate the effect that I want through flickity?

Sorry, not an easy way. Flickity does not support fade transitions. See hacks in #26
